contracts/ReaperAutoCompoundScreamLeverage.sol

```diff
@@ -527,15 +519,7 @@ contract ReaperAutoCompoundScreamLeverage is ReaperBaseStrategy {
             }
         }
 
-        uint256 withdrawAmount;
-
-        if (_useWithdrawFee) {
```

Comment:
removing anything to do with securityFee as per jeffery's request

Reply:
I am not sure we should potentially compromise our security to appease another protocol that may or may not make use of this. Are we really sure of the security implications of this? I think Beefy has 0 but they have harvest on deposit instead. I think their rule is if it harvests each deposit then you can set this to 0. Without being able to set it we would be defenseless if an attack proved feasible.